### PR TITLE
added Pulsar cluster deployment

### PR DESCRIPTION
**Pull request by tomusheq**

### CODE REVIEW

File: ansible/install_and_setup_pulsar.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        path: "/opt/pulsar/{{ item }}"
+        state: directory
+      loop:
+        - connectors
+    - name: Download Pulsar IO packages
+      get_url:
+        url: https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-{{ pulsar_version }}/connectors/pulsar-io-{{ item }}-{{ pulsar_version }}.nar
+        dest: /opt/pulsar/connectors/pulsar-io-{{ item }}-{{ pulsar_version }}.nar
+      loop:
+#        - aerospike
```
Comment (adamw) [inline ansible/install_and_setup_pulsar.yml]: can we remove these if they are not needed? (comments)

**Comment by adamw** [comment]
Are the .conf files standard, or are there any modificaitons?

File: ansible/install_and_setup_pulsar.yml (hunk 1 above)
Comment (tomusheq) [reply ansible/install_and_setup_pulsar.yml]: done

**Comment by tomusheq** [comment]
> Are the .conf files standard, or are there any modificaitons?

Configuration files are standard, without any modifications.

**Pull request merged**